src-java/flowhs-topology/flowhs-storm-topology/src/main/java/org/openkilda/wfm/topology/flowhs/fsm/update/actions/AllocatePrimaryResourcesAction.java

```diff
@@ -69,7 +82,9 @@ protected void allocate(FlowUpdateFsm stateMachine)
                 .forward(flow.getForwardPath())
                 .reverse(flow.getReversePath())
                 .build();
-        FlowPathPair newPaths = createFlowPathPair(flow, oldPaths, potentialPath, flowResources);
+        long overprovisionedBandwidth = getFlowsBandwidthSum(flowIds);
+        FlowPathPair newPaths = createFlowPathPair(flow, oldPaths, potentialPath, flowResources,
```

Comment:
It's a really controversial solution - you allow overprovisioning for the sum of all flows' bandwidth for a single new path, but can't guarantee that this bandwidth will be release by those flows as they can have different paths.

Reply:
Fixed.